docs/index.md

```diff
@@ -0,0 +1,136 @@
+---
+homepage: true
+layout: product
+includeInBreadcrumbs: true
+title: GOV.UK Prototype Components
+description: Common and experimental components that are not yet part of the GOV.UK Design System
+---
+
+GOV.UK Prototype Components contains the code you need to use common or experimental components that are not yet in the GOV.UK Design System.
+
+These components are currently experimental and more research is needed to validate them.
+
+| Component | Description |
+| - | - |
+| [xGovukAutocomplete](/autocomplete) | Implements the [Accessible autocomplete pattern](https://github.com/alphagov/accessible-autocomplete) to enhance a fixed list of options provided by a `<select>` element. |
+| [xGovukMasthead](/masthead) | Implements the masthead component [used on many GOV.UK product pages](https://github.com/alphagov/product-page-example). |
+| [xGovukPrimaryNavigation](/primary-navigation) | Implements the primary navigation component used on the GOV.UK Design System website. |
+| [xGovukRelatedNavigation](/related-navigation/) | Implements [related navigation component](https://components.publishing.service.gov.uk/component-guide/related_navigation) found in the `govuk_publishing_components` gem. |
+| [xGovukSubNavigation](/sub-navigation) | Implements the sub navigation component used on the GOV.UK Design System website. |
+
+Two JavaScript-only modules are also provided:
+
+| Module | Description |
+| - | - |
+| [Edge](/edge) | Define the edges of your prototype for research. |
+| [Warn on unsaved changes](/warn-on-unsaved-changes) | Warn users if they try to leave a page without saving changes to a form. |
+
+> [!NOTE]\
+> Prior to v1.0.0, this project included a collection of decorated form components. These can now be found in the [`govuk-decorated-components`](https://github.com/x-govuk/govuk-decorated-components) package.
+
+## Requirements
+
+Node.js v16 or later.
```

Comment:
I think this needs updating, as currently the package.json for this project has `"node": ">=18.0.0"` (which also needs to be updated to account for v20). Perhaps one for a separate PR?

Reply:
Opps, well spotted. Fixed the initial mis-match in #179.

Not sure if we should also update to minimum 20 or not? Are we dependent on a version of the prototype kit which depends on v20?